src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -3759,6 +3761,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return ' can't have any conflicts
             End If
 
+            ' Check no duplicate interfaces (ignoring tuple names)
+            Dim declaringSyntax = Me.GetDeclaringSyntaxNode(Of VisualBasicSyntaxNode)()
+            If declaringSyntax IsNot Nothing Then
+                Dim seenInterfaces = New Dictionary(Of TypeSymbol, TypeSymbol)(EqualsIgnoringComparer.InstanceIgnoringTupleNames)
+                For Each [interface] In interfaces
+                    Dim other As TypeSymbol = Nothing
+                    If seenInterfaces.TryGetValue([interface], other) Then
+                        Binder.ReportDiagnostic(diagnostics, declaringSyntax,
```

Comment:
>declaringSyntax [](start = 61, length = 15)

Consider if we can reuse logic from ReportInterfaceUnificationError to figure out location for the error.